main.py

```diff
@@ -1,10 +1,43 @@
-import init_django_orm  # noqa: F401
+import json
 
+import init_django_orm  # noqa: F401
+from django.db import transaction
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+    for player_data in players_data:
+        with transaction.atomic():
+            race_data = players_data[player_data]["race"]
```

Comment:
The access to `players_data[player_data]["race"]` is incorrect. Since `player_data` is an element of the list `players_data`, you should directly access `player_data["race"]` instead.